native_client/java/libdeepspeech/src/main/java/org/mozilla/deepspeech/libraryloader/utils/NativeLibraryLoader.java

```diff
@@ -0,0 +1,71 @@
+package org.mozilla.deepspeech.libraryloader.utils;
+
+import org.jetbrains.annotations.NotNull;
+
+import java.io.*;
+import java.net.URISyntaxException;
+import java.net.URL;
+
+/**
+ * Loads a given native libary
+ */
+public class NativeLibraryLoader {
```

Comment:
What ?!

Reply:
An object that loads native libraries. If somebody wants to put them into the jar, this loader makes that possible.

Reply:
And yet, we already package `libdeepspeech.so` in the AAR and in the JAR, and it works. Why do you need all that complicated stuff, then ?

Reply:
On Android it is not necessary to do that, as they are, as you already said, in the AAR. On desktop Java on the other hand, loading a native library that is packaged in the jar, is impossible without extracting it to a temporary location. The Android library loading configuration does not use this native library loader.

Reply:
How comes that `System.loadLibrary()` on Android is able to load from within the `AAR` ? It feels weird that they change the semantic of Java.